src/components/dropdown-menu/DropdownMenu.scss

```diff
@@ -54,13 +54,15 @@
                 box-shadow: $bdl-header-box-shadow;
             }
 
-            .submenu {
+            .submenu:not(.is-hidden) {
                 top: 100%;
                 right: 0;
+                bottom: auto;
                 left: auto;
 
-                &.is-hidden {
-                    display: none;
+                &.is-right-bottom-aligned {
+                    top: auto;
+                    bottom: 100%;
```

Comment:
I'm curious why `isRightBottomAligned` a better name when there doesn't seem to be any styles related to the horizontal position?

Reply:
This actually depends on the outer style